doxy

```diff
@@ -40,7 +38,14 @@ genPdf () {
         popd
         
         # move pdf to docs
-        mv /tmp/bill/${location}/latex/refman.pdf docs/${name}.pdf
+        mv ${tmp_location}/refman.pdf docs/${name}.pdf # TODO: Why is this refman.pdf?
```

Comment:
Sounds like a bug.

Reply:
Hold on, it generates refman.pdf. I have to auto rename it.

Reply:
Right.  Just leaving a comment that that makes no sense and I have found no way around it.  So the next confused person sees something.